app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -74,6 +68,7 @@ import org.oppia.android.domain.topic.PrimeTopicAssetsControllerModule
 import org.oppia.android.testing.AccessibilityTestRule
 import org.oppia.android.testing.TestLogReportingModule
 import org.oppia.android.testing.espresso.EditTextInputAction
+import org.oppia.android.testing.espresso.TextInputAction
```

Comment:
Suggesting to import the functions, not the class and check if Bazel successfully built after it or not

```suggestion
import org.oppia.android.testing.espresso.TextInputAction.Companion.hasErrorText
import org.oppia.android.testing.espresso.TextInputAction.Companion.hasNoErrorText
```

Reply:
Done